gdnative-sys/build.rs

```diff
@@ -164,7 +164,10 @@ mod header_binding {
                 format!(
                     "toolchains/llvm/prebuilt/{host}/sysroot/usr/include/{target_triple}",
                     host = &host_tag,
-                    target_triple = &target_triple,
+                    target_triple = match target_triple.as_str() {
```

Comment:
Since `target_triple` isn't used after this, I'd rather split this match into a separate `let` binding and switch to inline interpolation syntax for the whole `format!`:

```rust
let target_triple = match target_triple.as_str() {
    "armv7-linux-androideabi" => "arm-linux-androideabi", // Workaround for different naming scheme with NDK
    _ => &target_triple,
};

// snip

format!("toolchains/llvm/prebuilt/{host_tag}/sysroot/usr/include/{target_triple}")
```

Reply:
Works too for me I will do this and fix it using fmt